packages/@expo/cli/src/run/android/runAndroidAsync.ts

```diff
@@ -115,27 +125,18 @@ export async function runAndroidAsync(projectRoot: string, { install, ...options
     await uploadRemoteBuildCache(projectRoot, {
       platform: 'android',
       provider: projectConfig.exp.experiments?.remoteBuildCache.provider,
+      buildPath: options.binary,
     });
   }
 }
 
 async function installAppAsync(androidProjectRoot: string, props: ResolvedOptions) {
-  // Find the APK file path
-  const apkFile = await resolveInstallApkNameAsync(props.device.device, props);
-
-  if (apkFile) {
-    // Attempt to install the APK from the file path
-    const binaryPath = path.join(props.apkVariantDirectory, apkFile);
-    Log.log(chalk.gray`\u203A Installing ${binaryPath}`);
-    await props.device.installAppAsync(binaryPath);
```

Comment:
the installing log and props.device.installAppAsync call seem remove accidentally?

Reply:
This was already handled in 

https://github.com/expo/expo/blob/2e19a356f9d8da47379da071c9d3718acc3358ac/packages/@expo/cli/src/run/android/runAndroidAsync.ts#L99-L102